bnnvara/simple-bus-aws-bridge-bundle/1.1/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "bundles": {
+        "BNNVARA\\SimpleBusAwsBridgeBundle\\SimpleBusAwsBridgeBundle": ["all"],
+        "SimpleBus\\SymfonyBridge\\SimpleBusCommandBusBundle": ["all"],
+        "SimpleBus\\SymfonyBridge\\SimpleBusEventBusBundle": ["all"]
```

Comment:
This is not needed.

Reply:
This is needed for the package to work, otherwise it can't find the simplebus to handle the event.

Reply:
Flex will add this automatically. It is not needed.